zetaclient/chains/sui/observer/outbound.go

```diff
@@ -285,11 +285,6 @@ func parseGasUsed(tx models.SuiTransactionBlockResponse) (uint64, error) {
 		return 0, errors.Wrap(err, "storage rebate")
 	}
 
-	// should not happen
-	if (compCost + storageCost) < storageRebate {
```

Comment:
This seems not true when PTB transaction consumed multiple SUI coin objects and TSS gets a lot refund in this case. I think spending multiple SUI coin objects somehow reduced the storage usage.
![image](https://github.com/user-attachments/assets/4075b8cb-d4ec-457e-8171-0c03b9a82d62)